backend/webhooks/events/events.go

```diff
@@ -23,7 +21,7 @@ func StringIsValidEvent(value string) bool {
 func IsValidEvent(evt Event) bool {
 	var isValid bool
 	switch evt {
-	case User, UserCreate, UserUpdate, UserDelete, Email, EmailCreate, EmailPrimary, EmailDelete:
+	case "user", "user.update.email", UserCreate, UserUpdate, UserDelete, EmailCreate, EmailPrimary, EmailDelete:
```

Comment:
Why are `user` and `user.update.email` valid webhook events? When I create a webhook with e.g. `user` this never gets called, or am I missing something?

Reply:
it will. You suggested that if we trigger a webhook e.g. user.create that it should also be triggered for all people who subscribed to user (nested events). Did I misunderstand you on that one?

Reply:
Ohh sorry, I think I confused myself. You understood everything correct. But now as I am not confused anymore, I think I made a mistake by asking to remove the declaration of theses values https://github.com/teamhanko/hanko/pull/1291#discussion_r1465187247. They should be added back, so we have an easy to access list which events are supported.

Reply:
They are back and the option for disabling time based expiration is also implemented